nemo/core/classes/modelPT.py

```diff
@@ -902,17 +902,15 @@ def setup(self, stage: Optional[str] = None):
         if stage in ('fit', 'validate'):
             val_deferred_setup = (
                 'validation_ds' in self._cfg
-                and self._cfg.validation_ds is not None
+                and not self._cfg.validation_ds
```

Comment:
Why do we need this change? the change `not self._cfg.validation_ds` seems to be the opposite of the previous `self._cfg.validation_ds is not None`

Reply:
oh, sorry, the condition should be changed for `self.test_dataloader()`, as it is not None but [], corrected now